data/remnant/remnant missions.txt

```diff
@@ -3445,3 +3445,274 @@ event "remnant: pantica void sprites"
+			choice
+				`	"Have you asked someone senior for advice?"`
+				`	"Don't worry about it."`
+					goto worry
+				`	"Can I help with something?"`
+					goto help
+			`	"I did. The senior researcher told me that most of the team feels it. There are some concrete theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
+				goto help
+			label worry
+			`	"Thanks for the advice, but that is our purpose here - to worry about all the stuff that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
+			label help
+			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A*, and I could use a transport..."`
+			choice
+				`	"I don't know where that is."`
+					goto center
+				`	"Isn't that the center of the galaxy?"`
+					goto centeralt
+				`	"I have been there."`
+				`	"I have visited it a few times."`
+			label sagittarius
+			`	"Really?" She looks surprised. "I have seen sensor logs and recordings from previous visits, but not many people have actually been there." She pauses to take a bite from her neglected meal. "Being able to take part in these expeditions is something I have been looking forward to for ages, so of course Sagittarius A* is the first place on my list."`
+				goto center
+			label centeralt
+			`	"Yes, that is correct." She signs firmly.`
+			label center
+			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the Center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses, and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
+			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
+			choice
+				`	"Sure, what do you need?"`
+				`	"I'm busy right now, maybe later?"`
+					defer
+			`	"For this mission we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require 6 outfit space, and comes with a scientist, so you will need a free bunk for them, too.`
+			`	"Once the laboratory module is installed, it will simply be a matter of travelling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
+			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
```

Comment:
> "I am already packed and ready to go.

Reads weirdly. Perhaps: "I've already packed everything and am ready to go?"

Reply:
As a stylistic thing, the Remnant don't use contractions. Chalk it up to an idiosyncrasy in their language. But I'm open to adjusting the phrasing in other ways.